react/features/invite/components/AddPeopleDialog.web.js

```diff
@@ -27,6 +27,12 @@ class AddPeopleDialog extends Component {
      */
     static propTypes = {
         /**
+         * The {@link JitsiMeetConference} whcih will be used to invite "room"
+         * participants through the SIP Jibri (Video SIP gateway).
```

Comment:
Typo here with whcih. Also, more importantly, I have a question: a "room" as in a physical room?

Reply:
Hmm yes we use them for physical conference rooms, but maybe I should change the naming as it's more about 'video SIP gateway'